README.asciidoc

```diff
@@ -0,0 +1,401 @@
+//
```

Comment:
This wouldn't be copyright of Eclipse Foundation.

Can remove the entire copyright and notice section

Reply:
Done